luigi/tools/deps-tree.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from luigi.task import flatten
+from luigi.cmdline_parser import CmdlineParser
+import sys
```

Comment:
Can you copy in module documentation, with how the formatted tree will look like?